.github/workflows/ci.yml

```diff
@@ -10,11 +10,17 @@ jobs:
           - "2.7"
           - "3.0"
           - "3.1"
+          - "3.2"
         rails-version:
           - "7.0"
           - "6.1"
           - "6.0"
           - "main"
+        exclude:
+          - ruby-version: "2.7"
+            rails-version: "main"
```

Comment:
Could you point to where the latest version of Rails will drop support for Ruby `2.7`? The [Edge Guides](https://edgeguides.rubyonrails.org/getting_started.html) still [reference 2.7](https://edgeguides.rubyonrails.org/getting_started.html#installing-ruby).